addons/sys_list/XEH_postInit.sqf

```diff
@@ -8,6 +8,5 @@ if (!hasInterface) exitWith {};
 }, "", [DIK_CAPSLOCK, [true, false, true]]] call CBA_fnc_addKeybind;
 
 ["ACRE2", "OpenRadio", localize LSTRING(OpenRadio), {
-    [ACRE_ACTIVE_RADIO] call EFUNC(sys_radio,openRadio);
-    false
+    [ACRE_ACTIVE_RADIO] call EFUNC(sys_radio,openRadio)
```

Comment:
Don't we need a return value for addKeybind?

Reply:
`openRadio` returns `true`, I just made it consistent with other keybinds. Will test either way.